projects/packages/search/src/class-helper.php

```diff
@@ -848,6 +848,7 @@ public static function generate_initial_javascript_state() {
 			'overlayOptions'        => array(
 				'colorTheme'        => get_option( $prefix . 'color_theme', 'light' ),
 				'enableInfScroll'   => get_option( $prefix . 'inf_scroll', '1' ) === '1',
+				'enablePostDate'    => get_option( $prefix . 'post_date', is_multisite() ? '1' : '0' ) === '1',
```

Comment:
Thanks @jsnmoon! Testing this name change in my implementation.

> I wonder if it would make sense to always show the post date in the expanded search layout?

My concern was changing users experience in the update, without them actually hitting the toggle themselves. If this isn't a concern then it would allow us to remove the `is multisite` checks from the option and simplify things a bit?